modules/win-autoscale/variables.tf

```
@@ -264,3 +264,33 @@ variable "ToggleNewInstances" {
   description = "A/B toggle that forces a change to instance userdata, triggering new instances via the Autoscale update policy"
   default     = "A"
 }
+
+variable "AsgMetrics" {
+  type        = "string"
+  description = "(Optional) csv list of ASG metrics to collect. Must define EnableASGMetricsCollection to enable. Define MetricsCollectionGranularity and leave this option blank to collect all metrics"
+  default     = ""
+}
+
+variable "AsgSnsArn" {
+  type        = "string"
+  description = "(Optional) The Amazon Resource Name (ARN) of the Amazon Simple Notification Service (SNS) topic to send ASG events to. NOTE: Must be defined in conjunction with ASGNotificationTypes."
+  default     = ""
+}
+
+variable "AsgNotificationTypes" {
+  type        = "string"
+  description = "(Optional) A list of event types that trigger a notification. Event types can include any of the following types: autoscaling:EC2_INSTANCE_LAUNCH, autoscaling:EC2_INSTANCE_LAUNCH_ERROR, autoscaling:EC2_INSTANCE_TERMINATE, autoscaling:EC2_INSTANCE_TERMINATE_ERROR, and autoscaling:TEST_NOTIFICATION. NOTE: Must be defined in conjunction with ASGSNSARN."
+  default     = ""
+}
+
+variable "InstanceTerminationPolicies" {
+  type        = "string"
```

Use native types, join them in the tf module.

```suggestion
  type        = "list"
```